### PR TITLE
refactor(app): add explanation and link to cal card

### PR DESCRIPTION
**Pull request by sfoster1**
This will be an ever-present reminder of what calibration is that lives
in the calibration card.

### CODE REVIEW

**Comment by sfoster1** [comment]
Screenshot from 2020-11-19 12-25-39

**Review by emilywools — APPROVED** [review]
looks good!

File: app/src/components/RobotSettings/CalibrationCard.js [hunk 1]
```diff
@@ -189,6 +198,24 @@ export function CalibrationCard(props: Props): React.Node {
           {DOWNLOAD_CALIBRATION}
         </Link>
       </Flex>
+      <Box
+        borderBottom={BORDER_SOLID_LIGHT}
+        fontSize={FONT_SIZE_BODY_1}
+        padding={SPACING_3}
+      >
+        <Text display={DISPLAY_INLINE}>{CAL_EXPLANATION}</Text>
```
Comment (b-cooper) [inline app/src/components/RobotSettings/CalibrationCard.js:206]: This is the perfect example of a place where i18next would clean things up. It's fine for now, though in the spirit of avoiding `&nbsp;`'s you can always do it like so: 
```
   <Text>{`${CAL_EXPLANATION} `}</Text>
```

**Review by b-cooper — APPROVED** [review]
🧵

**Pull request merged**